cs-sitescope/src/main/java/io/cloudslang/content/sitescope/services/DeleteMonitorGroupService.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * (c) Copyright 2020 EntIT Software LLC, a Micro Focus company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.cloudslang.content.sitescope.services;
+
+import io.cloudslang.content.httpclient.entities.HttpClientInputs;
+import io.cloudslang.content.httpclient.services.HttpClientService;
+import io.cloudslang.content.sitescope.constants.Inputs;
+import io.cloudslang.content.sitescope.constants.SuccessMsgs;
+import io.cloudslang.content.sitescope.entities.DeleteMonitorGroupInputs;
+import io.cloudslang.content.sitescope.entities.SiteScopeCommonInputs;
+import io.cloudslang.content.sitescope.utils.HttpUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.client.utils.URIBuilder;
+import org.jetbrains.annotations.NotNull;
+
+import java.net.URISyntaxException;
+import java.util.Map;
+
+import static io.cloudslang.content.httpclient.build.auth.AuthTypes.BASIC;
+import static io.cloudslang.content.httpclient.entities.Constants.CHANGEIT;
+import static io.cloudslang.content.httpclient.entities.Constants.DEFAULT_JAVA_KEYSTORE;
+import static io.cloudslang.content.sitescope.constants.Constants.*;
+import static io.cloudslang.content.sitescope.constants.SuccessMsgs.DELETE_MONITOR_GROUP;
+import static io.cloudslang.content.sitescope.services.HttpCommons.setCommonHttpInputs;
+
+
+public class DeleteMonitorGroupService {
+
+    public @NotNull
+    Map<String, String> execute(@NotNull DeleteMonitorGroupInputs deleteMonitorGroupInputs) throws Exception {
+        final HttpClientInputs httpClientInputs = new HttpClientInputs();
+        final SiteScopeCommonInputs commonInputs = deleteMonitorGroupInputs.getCommonInputs();
+
+        setCommonHttpInputs(httpClientInputs, commonInputs);
+        httpClientInputs.setUrl(getUrl(deleteMonitorGroupInputs));
+        httpClientInputs.setQueryParamsAreURLEncoded(String.valueOf(true));
+        httpClientInputs.setAuthType(BASIC);
+        httpClientInputs.setUsername(commonInputs.getUsername());
+        httpClientInputs.setPassword(commonInputs.getPassword());
+        httpClientInputs.setMethod(DELETE);
+        httpClientInputs.setKeystore(DEFAULT_JAVA_KEYSTORE);
```

Comment:
this should be the inputs (if the inputs are empty we can use the defaultIfEmpty with this values)